apps/web/src/features/finance/actions/getPayments.ts

```
@@ -0,0 +1,643 @@
+  filters: any[] = [],
+  page = 0,
+  pageSize = 25,
+  sorting: any[] = []
+) {
+  try {
+    const supabase = await createClient();
+
+    // Use the view for simpler filtering
+    let query = supabase
+      .from("payments_with_details")
+      .select(`*`, { count: "exact" });
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)